internal/pkg/addon/package_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package addon
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+)
+
+type addonMocks struct {
+	// uploader *mocks.Mockuploader
+	// ws *mocks.MockworkspaceReader
+}
```

Comment:
nit: would you mind deleting the dead-code in the test file (although it'll be populate soon)?